upgrading/migrating_etcd.adoc

```diff
@@ -64,6 +64,14 @@ Embedded etcd to External etcd].
 [[etcd-data-migration-automated]]
 == Running the Automated Migration Playbook
 
+[IMPORTANT]
+====
+If the migration playbooks fail before the masters are reconfigured to support 
+etcd v3 storage, you must
+xref:etcd-data-migration-recovering[roll back] the migration process.
```

Comment:
@ahardin-rh, I shortened the first link to just [roll back] because it was hard to differentiate the sequential links.

Reply:
Looks good!